iron-location.html

```diff
@@ -135,34 +139,54 @@
           value: false
         }
       },
+
       hostAttributes: {
         hidden: true
       },
+
       observers: [
         '_updateUrl(path, query, hash)'
       ],
+
       attached: function() {
+        this.initialize();
+      },
+
+      ready: function() {
+        if (!Polymer.version || Polymer.version >= '2') {
```

Comment:
Also, why this cannot be done on `ready` also for Polymer 1.x? Could  you add a comment for the future maintaner? :)